<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSerialTestForm</class>
 <widget class="QWidget" name="DSerialTestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">DSerialTestForm</string>
  </property>
  <layout class="QVBoxLayout" name="serialPageLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_serial">
     <property name="title">
      <string>SerialportDebug</string>
     </property>
     <layout class="QVBoxLayout" name="groupBox_verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QWidget" name="serial_head_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_serial">
           <property name="text">
            <string>serial:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_port">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_refresh">
           <property name="text">
            <string>refresh</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_baud"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_open">
           <property name="text">
            <string>open</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn_receive_ascii">
           <property name="text">
            <string notr="true">ASCII</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn_receive_hex">
           <property name="text">
            <string notr="true">Hex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_clear">
           <property name="text">
            <string>clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item>
       <widget class="QWidget" name="serial_send_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_send">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>send:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_send"/>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn_send_ascii">
           <property name="text">
            <string notr="true">ASCII</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn_send_hex">
           <property name="text">
            <string notr="true">Hex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_send">
           <property name="text">
            <string>send</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
